mlflow/langchain/runnables.py

```diff
@@ -309,16 +312,14 @@ def _save_runnable_with_steps(model, file_path: Union[Path, str], loader_fn=None
         # Save each step into a subfolder named by step
         save_runnable_path = steps_path / step
         save_runnable_path.mkdir()
-        if result := _save_internal_runnables(runnable, save_runnable_path, loader_fn, persist_dir):
-            steps_conf[step] = result
+        result = _save_internal_runnables(runnable, save_runnable_path, loader_fn, persist_dir)
+        if isinstance(result, Exception):
+            unsaved_runnables[step] = f"{runnable} -- {result}"
```

Comment:
can we use try-catch?

Reply:
yeah but can we use try-catch for detecting errors?

Reply:
Follow-up: https://github.com/mlflow/mlflow/pull/11516